app/(server)/_features/invitee/schema.ts

```diff
@@ -0,0 +1,15 @@
+import { serial, text, pgTable, timestamp } from 'drizzle-orm/pg-core';
+import { type InferSelectModel, sql } from 'drizzle-orm';
+
+export const invitees = pgTable('invitees', {
+  id: serial('id').primaryKey(),
+  email: text('email').notNull().unique(),
+  name: text('name').notNull(),
+  whitelistFeature: text('whitelist_feature')
+    .array()
+    .notNull()
+    .default(sql`array[]::text[]`),
+  createdAt: timestamp('created_at').notNull().defaultNow(),
+});
```

Comment:
I still don't know whether `invitees` is a good table name. I was thinking about something else such as `guests`, `unregistered_users`, but after thinking a bit I chose `invitees` which represents a list of invited users. What do you think @GagahGK ?

Reply:
@faiq-naufal 

I think invitees is a good name but, i think we still need an added context, to tell which invitees is the table meant for.

how about ? `early_invitees` or maybe `account_invitee` 
`account_invitee` since it meant for account creation

while early for early access feature

Reply:
@GagahGK  What about `early_access_invitees`? This name is longer, but it provides more context. `account_invitee` is a good name, but I'm afraid it can be translated as "an existing user invites another unregistered user using email".

Reply:
sure I think early_access_invitee or early_invitee would be a better name

Reply:
@GagahGK  I updated the table to `early_access_invitees` on this commit [6f7e927](https://github.com/inlivedev/inlive-room/pull/143/commits/6f7e92783b019b3997fa56058f0cd0c417ca5209). Invitee is a singular, and invitees is plural